bin/build

```diff
@@ -2,26 +2,47 @@
 
 set -e
 
-. bin/build_utils
+# Use Docker BuildKit to efficiently build the targets.
+# With BuildKit, only relevant layers for the target are built.
+export DOCKER_BUILDKIT=1
+DEBUG=false
 
-FULL_VERSION_TAG="$(full_version_tag)"
+if [ "$1" == "--debug" ]; then
+    DEBUG=true
+fi
 
-echo "---"
+# go to repo root folder for execution
+cd $(dirname $0)/..
```

Comment:
how come we need this now whereas before we didn't?

Reply:
Who said we didn't?
The script cannot be run from any other location besides from the root folder using `./bin/build`.
I wanted to allow it to run from any location; why it should care where is it called from?

Reply:
ah i see. I don't see the need to run it from anywhere else than the root folder but if you think there is such a need then i'm ok with it if this is the only change. 

From which location did you need to run it other than the root folder?

Reply:
I tried from `bin` folder and failed.
Short investigation brought this solution, which allows running from any location.